openhtf/io/output/print_ToConsole.py

```diff
@@ -0,0 +1,89 @@
+"""Module to display test summary on console."""
+import os
+import sys
+from openhtf import util
+from openhtf.io import test_record
+from openhtf.util import measurements as mea
+import datetime
+class PrintToConsole():
+  """Print test results with failure info on console. """
+
+  def __init__(self):
+    if os.name == 'posix':    #Linux and Mac
+      self.RED =  '\033[91m'
+      self.GREEN =  '\033[92m'
+      self.ORANGE = '\033[93m'
+      self.RESET =  '\033[0m'
+      self.BOLD =  '\033[1m'
+    else: 
+      self.RED    = ""
+      self.GREEN  = ""
+      self.ORANGE = ""
+      self.RESET  = ""
+      self.BOLD   = ""
+
+    self.color_table = {
+        'PASS':self.GREEN,
+        'FAIL':self.RED,
+        'ERROR':self.ORANGE
+  }
+  # pylint: disable=invalid-name
+  def __call__(self, record):
+    indent2 = ''.join(('\n', ' '*2))
+    indent4 = ''.join(('\n', ' '*4))
+  
+    status = str(record.outcome)[-5:]
+    if status[0] == '.':
+      status = status[1:]
+
+    color = self.color_table[status]
+
+    output_lines = []
+    msg = ''.join((os.path.basename(sys.argv[0]), ': ', status))
+    msg = ''.join((color, self.BOLD, msg, self.RESET))
+    
+    output_lines.append(msg)
+
+    if status == 'FAIL' or status == 'ERROR':      
```

Comment:
I thought the plan was to pull these out of the appropriate enums instead of hard-coding which strings to check for.

Also:
```
if status in (test_record.Outcome.FAIL, test_record.Outcome.ERROR):
```